packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -0,0 +1,244 @@
+        nextCol = this.props.rows[0].cells.length - 1;
+        if (event.ctrlKey) {
+          nextRow = (this.props.rows.length - 1) + this.getNumberOfHeaderRows();
+        }
+        break;
+      default:
+        return;
+    }
+    if (nextRow >= this.grid.rows.length || nextCol >= this.grid.rows[0].cells.length) {
+      event.preventDefault();
+      return;
+    }
+    if (nextCol < 0 || nextRow < 0) {
+      event.preventDefault();
+      return;
+    }
+
+    // Remove the ability to receive visible focus as only one cell should be able to receive tab focus.
+    // This is also how tabbing back to the grid ensures the grid last cell with the focus will be re-entered.
+    this.grid.rows[currentFocussedRow].cells[currentFocussedCol].setAttribute('tabIndex', '-1');
+
+    this.focussedRow = nextRow;
+    this.focussedCol = nextCol;
+
+    const elementWithTabStop = this.grid.rows[nextRow].cells[nextCol];
+    elementWithTabStop.setAttribute('tabIndex', '0');
+    elementWithTabStop.focus();
+    event.preventDefault();
+  }
+
+  handleClick(event) {
+    // Is this event for something that is on the grid
+    if (event.target?.parentNode?.parentNode?.parentNode?.id !== this.props.id) {
+      return;
+    }
+    // Remove Tab stop from previous cell in table that has focus and set it to the cell that was clicked.
+    this.grid.rows[this.focussedRow].cells[this.focussedCol].setAttribute('tabIndex', '-1');
+    this.focussedRow = event.target.parentElement.rowIndex;
+    this.focussedCol = event.target.cellIndex;
+    const clickedCell = event.target;
+    clickedCell.setAttribute('tabIndex', '0');
+    clickedCell.focus();
+    event.preventDefault();
+  }
+
+  getNumberOfHeaderRows() {
+    return this.props.columnHeaders?.pinnedColumns?.length > 0 ? 1 : 0;
+  }
+
+  getCellData(cell, cellColumnIndex) {
+    const cellClassName = { className: classNames(cx(this.props.rowHeaderIndex === cellColumnIndex ? 'worklist-data-grid-row-header' : 'worklist-data-grid-cell-data')) };
+    const tabIndex = { tabIndex: '-1' };
+    return this.props.rowHeaderIndex === cellColumnIndex ? (<th key={cellColumnIndex} {...tabIndex} {...cellClassName} role="rowheader">{cell.cellContent}</th>) : (<td key={cellColumnIndex} {...tabIndex} {...cellClassName}>{cell.cellContent}</td>);
+  }
+
+  buildColumnHeaders(colHeader) {
+    if (colHeader?.pinnedColumns?.length > 0) {
+      return (
+        <tr height={this.props.columnHeaderHeight}>
+          {colHeader.pinnedColumns.map(columnData => (this.buildSingleColumnHeading(columnData)))}
+        </tr>
+      );
+    }
+    return undefined;
+  }
+
+  buildSingleColumnHeading(columnData) {
+    const width = columnData.width || this.props.columnHeaderDefaultWidth;
+    const height = columnData.height || this.props.columnHeaderHeight;
+    const columnHeaderClassName = classNames(cx('worklist-data-grid-column-header'));
+    return (
+      /* eslint-disable react/forbid-dom-props */
```

Comment:
Why are we disabling the linter here?

Reply:
We had to disable it because the linter expects us not to pass props to DOM nodes. But we needed to provide the style. Also, we had noticed that this particular rule has been disabled several times in our code base. So, we followed existing precedent.